.env

```diff
@@ -18,6 +18,7 @@ DOMAIN_PROD=www.example.com
 DB_VERSION=9.6
 DB_USER=odoo
 DB_PASSWORD=odoopassword
+DBFILTER=.*
```

Comment:
I'd use `DB_FILTER` instead, just to keep it consistent.

Reply:
Hi

This not work for me with 
DBFILTER = "^%d$"

Can you help me please